release-notes/5.0/5.0.0-preview3.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.0.0-preview3 released 15 June 2022
+
```

Comment:
Should we list Encrypt  changes as a breaking change? We have added a new value "Strict", "Optional" and "Mandatory".